beancount/scripts/format_test.py

```diff
@@ -244,6 +244,44 @@ def test_arithmetic_expressions(self, filename):
 
         """), result.stdout)
 
+    @test_utils.docfile
+    def test_parenthesized_binary_expressions(self, filename):
+        """
+        2016-08-01 open Expenses:Test
+        2016-08-01 open Assets:Test
+
+        2016-08-02 * "" ""
+          Expenses:Test     10.0 USD
+          Expenses:Test (10.0/2) USD
+          Expenses:Test (10.0+2) USD
+          Expenses:Test (10.0-2) USD
+          Expenses:Test (10.0*2) USD
+          Expenses:Test (-10.0*+2) USD
+          Expenses:Test (-1,0.0*+2.0) USD
```

Comment:
This looks like a funny number. Updating the regexp to reject these may be overkill, but I would at least use a proper thousands separator in the example, ie `-1,000.00` instead than `-1,0.0`.

Reply:
> Updating the regexp to reject these may be overkill

Especially since different cultures use different separators (e.g: [the Indian numbering system](https://en.wikipedia.org/wiki/Indian_numbering_system)) :-).

I can update the test however.